README.md

```diff
@@ -43,17 +40,49 @@ Using the command line:
   - Invoke "Test" target from "Build.proj". Likely, you need to build test project first, so put in "build" target as well. 
         *msbuild build.proj /t:build;test /p:scope=ResourceManagement\Compute*
 
-### To on-board new libraries
-Follow existing library and create a new folder under "ResourceManagement".
-  - Note: To simplify test discovery, the test folder must be named with ".test" or ".tests"
-  
-If for platform reasons that your library won't use NetCore project system, 3 notes
-  - In your library csproject file, set the msbuild property "AutoRestProjects" to "true"
-  - In your test project files, set both "AutoRestProjects" and "SDKTestProject" to "true"
-  - To simplify test discovery, the test folder must be named with ".tests"
+## To on-board new libraries
+
+### Project Structure
+
+In "src\ResourceManagement", you will find packages for services that have already been implemented
+
+  - Each service contains a package for their generated code, as well as a package for tests
+  - The file 'generate.cmd', used to generate library code for the given service, can also be found in these packages
+
```

Comment:
The Folder 'Generated' contains generated code, The Folder 'Customizations' contains additions to the generated code - this can include additions to the genearetd partial classes, or additional classes that augment the SDK or call generated code.